apps/nextjs-website/src/lib/strapi/tutorial.ts

```diff
@@ -38,7 +39,6 @@ export const TutorialCodec = t.strict({
       locale: t.string,
       bannerLinks: t.union([NullToUndefinedCodec, t.array(BannerLinkCodec)]),
       relatedLinks: t.union([NullToUndefinedCodec, RelatedLinksCodec]),
```

Comment:
If u accet previous suggestion you should re-add product
```suggestion
      relatedLinks: t.union([NullToUndefinedCodec, RelatedLinksCodec]),
      product: t.strict({ data: ProductCodec }),
```